book/coloring_and_theming.md

```diff
@@ -517,3 +517,20 @@ $env.config = {
   ...
 }
 ```
+
+## Accessibility
+
+It's often desired to have the minimum amount of decorations when using a screen reader. In those cases, it's possible to disable borders and other decorations for both table and errors with the following options:
+
+```shell
+# in $nu.config-file
```

Comment:
```suggestion
# in $nu.config-path
```
:wink:

Reply:
ooh there were more `$nu.config-file` in that page :scream: 
thanks for fixing the other ones :pray:

Reply:
No problem! They were the reason I wrote that in the first place 😌